prboom2/src/m_menu.c

```diff
@@ -2887,9 +2887,6 @@ static const char *videomodes[] = {
   "OpenGL",
   NULL};
 
-static const char *gltexformats[] = {
```

Comment:
I assume tex format options used to be in the menu and were removed

Reply:
I believe that's correct